appcues/consumer-rules.pro

```diff
@@ -0,0 +1,5 @@
+-keepclassmembers class com.appcues.data.remote.response.** {
+  <init>(...);
+  <fields>;
+  <methods>;
+}
```

Comment:
maybe can simplify to 
```
-keep class com.appcues.data.remote.response.** { *; }
```
to keep everything in the JSON model package

Reply:
I will try, thanks